bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/dto/ActionDTOMapper.java

```diff
@@ -1,11 +1,10 @@
 /**
- * Copyright (c) 2016 Markus Rathgeb
+ * Copyright (c) 1997, 2016 by ProSyst Software GmbH and others.
```

Comment:
Note that under these circumstances, the PR with this header shouldn't have been merged in the first place, because no individuals are meant to be listed in the header, but in the author tag. The headers always have to carry "and others", such that any contributor is covered by it.